app/jobs/itms_job.rb

```diff
@@ -183,4 +198,11 @@ def find_or_create_additional_info_by_json(json, scope)
       scope.find_or_create_by!(nazov: j['nazov'], url: j['url'])
     end
   end
+
+  def latest_timestamp
```

Comment:
tomuto kodu by som sa vyhol, lepsie je:

- ist priamo `modifiedSince: Entity...`
- alebo mat ten model v `cattr` (ak sa pouziva aj inde) a potom `modifiedSince: model...`

ale tu uz sa nieco podobne riesi inak - https://github.com/slovensko-digital/harvester.ecosystem/pull/60/files#diff-210e335c929186a602a75dc2b96015abR63 tak by som sa drzal toho (nech sa to pri pripadnej uprave da sfuknut naraz)

takze by som cez `define_method` pridal `latest_#{base_name.underscore}_timestamp`